tests/test_plot_utils.py

```diff
@@ -0,0 +1,32 @@
+from inspect_evals.metadata import HumanBaseline
+from src.plots.plot_utils import (
+    create_hover_text,
+    get_human_baseline,
+    get_provider_color_palette,
+)
+
+
+def test_human_baseline(eval_logs):
+    log = eval_logs[0]
+    assert get_human_baseline(log) is None
+    setattr(
+        log.task_metadata,
+        "human_baseline",
+        HumanBaseline(metric="whatever", score=42, source="http://example.com/"),
+    )
+    assert get_human_baseline(log) == 42
+
+
+def test_get_provider_color_palette():
+    providers = {"A", "B"}
```

Comment:
Can you check whether the color palette is the same if the order of the providers is different? It should sort the providers so it should assign the same colors.

Reply:
`set` is an unordered collection so changing an order gives the same result:

```
>>> list({"A", "B"})
['A', 'B']
>>> list({"B", "A"})
['A', 'B']
```

The same is true if you insert the elements manually.